core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -59,6 +62,7 @@ class EmbeddedZookeeper() extends Logging {
 
     Iterator.continually(isDown()).exists(identity)
 
+    txnLog.close()
```

Comment:
I think you can just call `zookeeper.getTxnLogFactory.close()`

Reply:
It doesn't matter much; it will be the same obj. ref anyway